src/styles/_catalog.scss

```diff
@@ -1,10 +1,21 @@
+@mixin columns($width, $col) {
+  @media (min-width: $width) {
+    grid-template-columns: repeat($col, 200px);
+  }
+}
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
```

Comment:
that's how cards look on smaller breakpoints, you should center them:
![image](https://user-images.githubusercontent.com/37161315/170264862-05551cdf-3898-4478-a3b4-4f218ee8ede4.png)